src/clr-angular/utils/_theme.dark.clarity.scss

```diff
@@ -624,6 +624,7 @@ $clr-stack-view-stack-block-expanded-bg-color: #324f61;
 $clr-stack-view-stack-block-expandable-hover: #324f61;
 $clr-stack-view-stack-block-content-text-color: #adbbc4;
 $clr-stack-view-stack-block-expanded-text-color: $clr-white;
+$clr-stack-view-stack-block-caret-color: $gray-dark-midtone;
```

Comment:
Wait, is this the correct color for the dark theme? I'd expect it to be lighter.

Reply:
I checked. We are using the same color for the icon. We didn't replace the generated icon in dark theme.